src/mlpack/core/optimizers/parallel_sgd/decay_policies/constant_step.hpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file constant_step.hpp
+ * @author Shikhar Bhardwaj
+ *
+ * Constant step size policy for parallel Stochastic Gradient Descent.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_PARALLEL_SGD_CONSTANT_STEP_HPP
+#define MLPACK_CORE_OPTIMIZERS_PARALLEL_SGD_CONSTANT_STEP_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace optimization {
+
+/**
+ * Implementation of the ConstantStep stepsize decay policy for parallel SGD.
+ */
+class ConstantStep
+{
+ public:
+  ConstantStep(double initalStep) : step(initalStep) { /* Nothing to do */ }
```

Comment:
Can you add a comment for the constructor and the parameter? Also do you think you could use ``const double initalStep`` here, to be consistent with the rest of the codebase?

Not sure, can we find a reasonable default step value?